### PR TITLE
Update CSRanking with 2019 Fall UMass Lowell professors

### PR DESCRIPTION
**Pull request by jeapostrophe**
This letter from our dean mentions all the new hires:

[Welcome Back_Fall 2019.pdf](https://github.com/emeryberger/CSrankings/files/3611626/Welcome.Back_Fall.2019.pdf)

I think the biggest controversy might be that Hadi Amiri doesn't start until Winter.

Guanling Chen, btw, is removed because he has started a company and left the university.

Finally, Xinwen Fu has moved back to UML from UCF.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -5496,7 +5496,6 @@ Guangyan Zhang,Tsinghua University,http://storage.cs.tsinghua.edu.cn/~zgy,NOSCHO
 Guangyu Sun 0003,Peking University,http://ceca.pku.edu.cn/en/team.php?action=show&member_id=15,m3f70oYAAAAJ
 Guangzhi Qu,Oakland University,http://www.secs.oakland.edu/~gqu,Etw6t2kAAAAJ
 Guanhua Yan,Binghamton University,https://www.binghamton.edu/cs/people/ghyan.html,6rPWxsgAAAAJ
-Guanling Chen,University of Massachusetts Lowell,http://www.cs.uml.edu/~glchen,-4aVBKAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:5499]: Please move this entry to `old/industry.csv` (and add the name of the company as a final field, if known; if not, please put TBD).

Comment (emeryberger) [reply csrankings.csv:5499]: Also, the link in your post is unreadable.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please address the move by adding to the file as directed. If you can point to a public document for the letter your link includes (which I cannot read), that'd be helpful.

**Comment by jeapostrophe** [comment]
I update the industry database and have updated the link in the description. Thanks for looking at this so quickly!

**Pull request merged**